api/camunda/camunda-openapi.yaml

```diff
@@ -2113,9 +2113,12 @@ paths:
       tags:
         - Role
       operationId: createRole
-      summary: Create role
+      summary: Create role (Work-in-Progress)
       description: |
         Create a new role.
+
+        This API is in a Work-in-Progress state and will undergo potential breaking changes till
```

Comment:
I'll follow up with an upstream PR to correct the preposition "till" -> "until" on all of these.